docs/index.md

```diff
@@ -1087,16 +1087,30 @@ Files specified using `--file` _are not affected_ by this option.
 
 Can be specified multiple times.
 
-### `--extension <ext>, --watch-extensions <ext>`
-
-> _Updated in v6.0.0. Previously `--watch-extensions`, but now expanded to affect general test file loading behavior. `--watch-extensions` is now an alias_
+### `--extension <ext>`
 
 Files having this extension will be considered test files. Defaults to `js`.
 
-Affects `--watch` behavior.
-
 Specifying `--extension` will _remove_ `.js` as a test file extension; use `--extension js` to re-add it. For example, to load `.mjs` and `.js` test files, you must supply `--extension mjs --extension js`.
 
+### `--watch-files <file|directory|glob>`
+
+> _New in v7.0.0_
+
+List of comma separated paths or globs to watch when `--watch` is set. If a file matching the given glob changes or is added or removed mocha will rerun all tests.
```

Comment:
Our main format for `array` typed options is repeating the flag. In some cases we also allow comma separated lists of values.
Therefore I prefer: "List of paths or globs to watch [...]"
then please add: "--watch-files can accept a comma-delimited list; --watch-files ... is equivalent to --watch-files ..."

Reply:
https://github.com/mochajs/mocha/pull/3980/commits/b422fa8f8ecc2d01beecc62e72f7463c2dba8075